src/private/privateAPIs.ts

```diff
@@ -203,3 +204,18 @@ export function getConfigSetting(callback: (value: string) => void, key: string)
 
   sendMessageToParent('getConfigSetting', [key], callback);
 }
+
+/**
+ * @private
+ * register a handler to be called when a user setting changes. The changed setting key & value is provided in the callback.
+ * @param settingKeys List of user setting changes to subscribe
+ * @param handler When a subscribed setting is updated this handler is called
+ */
+export function registerUserSettingsChangeHandler(
+  settingKeys: UserSettingKeys[],
+  handler: (settingKey: UserSettingKeys, value: any) => void,
+): void {
+  ensureInitialized(FrameContexts.content);
```

Comment:
This API should work in any frame context so you probably shouldn't pass a frame context to the `ensureInitialized` call.

Reply:
This is done